### DISCUSSION TITLE
Global Human Settlement Layer 2023 - Load all epochs from 1975 through 2030

### DISCUSSION THREAD

**Discussion by FlorinZai** [samapriya/awesome-gee-community-datasets#142] [General]
Hello Samapriya, 
Thank you very much for this amazing catalog with so many useful data available now in EE.

I am trying to access all the epochs (1975-2030) of the GHS 2023 dataset  https://gee-community-catalog.org/projects/ghsl/?h=ghs#earth-engine-snippet
For example the GHS-BUILT-S R2023A
var GHS_BUILT_S_2030 = ee.Image("projects/sat-io/open-datasets/GHS/GHS_BUILT_S_E2030_GLOBE_R2023A_54009_100_V1_0");
Using the suggested code only the 2030 loads. Changing the year reveals that the other epochs are not available. Same for the PoP dataset.
Is it available with a different name, or will you upload everything at a later date?

Thanks,
Florin

**Comment by samapriya** [comment]
This was fixed and the datasets were updated
